index.html

```diff
@@ -265,11 +265,11 @@ <h2 class="title2">uber navigation</h2>
        Uber
       </p>
    
-    <img src="Images./Counter.png" class="round" alt="dot">
+    <img src="Images./Counter (1).png" class="round" alt="dot">
       
       <p class="role">Lead developer</p>
```

Comment:
- [ ] As stated by the previous reviewer try to make your Portfolio responsive. Below you can see that there are empty margins when the screen size grows😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220322139-9ae1fce4-0394-49d1-90d5-379afa9ba09d.png)  |     ![Recording 2023-02-21 at 12 35 28](https://user-images.githubusercontent.com/83360636/220321725-4f704d61-3853-402f-8182-497771bb7923.gif) |